### PR TITLE
Added German translation

### PR DESCRIPTION
**Pull request by guerda**
I created the German translation for MozStumbler based on the English version.

### CODE REVIEW

File: res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<resources>
+
+    <string name="app_name">MozStumbler</string>
+    <string name="action_about">Über MozStumbler</string>
+    <string name="action_preferences">Einstellungen</string>
+    <string name="mozilla_title">Mozilla</string>
+    <string name="start_scanning">Starte Scan-Vorgang</string>
+    <string name="stop_scanning">Stoppe Scan-Vorgang</string>
+    <string name="view_leaderboard">Öffne Bestenliste</string>
+    <string name="view_map">Test Mozilla Location Service</string>
```
Comment (hannosch) [inline res/values-de/strings.xml]: Shouldn't this be something like "Teste Lokalisierungsdienst" or at least replace "Test" with "Teste".

File: res/values-de/strings.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <string name="reports_sent">Gesendete Berichte: %1$d</string>
+    <string name="service_scanning">Erfassen von WLAN und Funkzellen</string>
+
+    <string name="enter_nickname">Gebe Spitznamen ein (optional)</string>
+    <string name="enter_nickname_title">Spitzname: </string>
+    <string name="enter_nickname_longtext">Dein Spitzname taucht in der Bestenliste auf</string>
+
+    <string name="battery_low_warning">Akkustand niedrig, MozStumbler stoppt Erfassung</string>
+
+    <string name="update_title">MozStumbler Update herunteladen?</string>
```
Comment (hannosch) [inline res/values-de/strings.xml]: What about "Neue MozStumbler Version herunterladen?" - and there's a typo in "herunteladen".

**Comment by guerda** [comment]
Hi Hanno!

I updated the translation, thank you for the hints! I hope, now it's more fitting.

**Comment by cpeterso** [comment]
@guerda: thanks for the translation. @hannosch recommended that I merge the German translation in PR #215, but please feel free to file a new PR for improved translations or new strings.

**Pull request closed without merging**